include/lttng/ust-events.h

```diff
@@ -110,7 +110,12 @@ struct lttng_enum_value {
 struct lttng_enum_entry {
 	struct lttng_enum_value start, end; /* start and end are inclusive */
 	const char *string;
-	char padding[LTTNG_UST_ENUM_ENTRY_PADDING];
+	union {
+		struct {
+			unsigned int is_auto:1;
```

Comment:
How about an `unsigned int` with option bits? I don't want to waste 7 bits, it's too hard.